src/components/common/calendar/Calendar.js

```diff
@@ -154,6 +159,7 @@ function CalendarApp() {
         visible={isModalVisible}
         onOk={handleOk}
         onCancel={handleCancel}
+        className="calendar-event-modal"
```

Comment:
Good job with the renaming of the button. Luis does bring up a good thought about the renaming.